Gemfile.lock

```diff
@@ -175,13 +175,13 @@ GEM
     childprocess (4.1.0)
     coderay (1.1.3)
     concurrent-ruby (1.1.10)
-    connection_pool (2.2.5)
+    connection_pool (2.3.0)
     crass (1.0.6)
-    cssbundling-rails (1.1.0)
+    cssbundling-rails (1.1.1)
       railties (>= 6.0.0)
-    debug (1.5.0)
+    debug (1.6.2)
```

Comment:
### [debug](https://github.com/ruby/debug)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.5.0 → 1.6.2

[_~~change-log~~, [source-code](https://github.com/ruby/debug), [gem-diff](https://my.diffend.io/gems/debug/1.5.0/1.6.2)_]





<details>
<summary>Commits</summary>

A change of **137** commits. See the full changes on [the compare page](https://github.com/ruby/debug/compare/v1.5.0...v1.6.2).

These are the first 10 commits:
- (9d19068) [Assume loopback server address is localfs](https://github.com/ruby/debug/commit/9d19068e2f5bca2f9cdc02b3a76c9a3dda81fd12)
- (5d16d87) [revert last commit](https://github.com/ruby/debug/commit/5d16d8731e85ae7ef20440d7c8e12fa83bef1995)
- (1fce258) [skip feature check on MRI](https://github.com/ruby/debug/commit/1fce2583d1e71419998507898ba5f7eea815133f)
- (5edf769) [remove `bv` experimental command](https://github.com/ruby/debug/commit/5edf7694f80688e40c8afa8139951c71c451c097)
- (63a252b) [Print information that protocol tests are skipped only once](https://github.com/ruby/debug/commit/63a252bc40d2c4f64aa08c5cfd9362405f18a1c6)
- (1a4da7c) [Add send_dap_request to include dap response verification](https://github.com/ruby/debug/commit/1a4da7c7dafb19917560cd3b6315d69d69c1f7cc)
- (35de32f) [Add send_cdp_request to include cdp response verification](https://github.com/ruby/debug/commit/35de32f683ac0284a191ef8bb52d5b259868c6e4)
- (d20cc8a) [Rename assert_disconnect_result to cleanup_reader](https://github.com/ruby/debug/commit/d20cc8af2c72bccdbd00b260e42c4e3ae6eec455)
- (4b2219b) [Cleanup reader for both DAP/CAP termination](https://github.com/ruby/debug/commit/4b2219bf1600cd595ae7ae37fbbd9340ab57c7e5)
- (33d9c25) [CheckBreakpoint should only stop at the first condition fulf…](https://github.com/ruby/debug/commit/33d9c2528d18399271fb47542c27469fb215838e)


</details>